<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1616</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_HY">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_HY1">
        <property name="title">
         <string>Hydrophone #1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>time domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>frequency domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="10">
            <widget class="QPushButton" name="pushButton_HY1_endPlot">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="11">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Record:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_HY1_ymin"/>
           </item>
           <item row="1" column="9">
            <widget class="QPushButton" name="pushButton_HY1_startPlot">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>tick</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_HY1_ymax"/>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="pushButton_HY1_minmax">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="1" column="14">
            <widget class="QPushButton" name="pushButton_HY1_save">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="1" column="13">
            <widget class="QPushButton" name="pushButton_HY1_endRec">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_41">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_max</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_40">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_min</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Plot:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QLineEdit" name="lineEdit_HY1_tick"/>
           </item>
           <item row="1" column="12">
            <widget class="QPushButton" name="pushButton_HY1_startRec">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="14">
            <widget class="QComboBox" name="comboBox_HY1"/>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget_HY1_freq" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_HY1_time" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_HY1_2">
        <property name="title">
         <string>Hydrophone #2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="1" colspan="2">
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="1">
            <widget class="QLabel" name="label_55">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_min</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="pushButton_HY2_minmax">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="1" column="12">
            <widget class="QPushButton" name="pushButton_HY2_startRec">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_56">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_max</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="9">
            <widget class="QPushButton" name="pushButton_HY2_startPlot">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_HY2_ymax"/>
           </item>
           <item row="1" column="10">
            <widget class="QPushButton" name="pushButton_HY2_endPlot">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>tick</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QLineEdit" name="lineEdit_HY2_tick"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_HY2_ymin"/>
           </item>
           <item row="1" column="13">
            <widget class="QPushButton" name="pushButton_HY2_endRec">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="11">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Record:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Plot:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="14">
            <widget class="QPushButton" name="pushButton_HY2_save">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="14">
            <widget class="QComboBox" name="comboBox_HY2"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_HY2_time" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget_HY2_freq" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_57">
           <property name="text">
            <string>time domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_58">
           <property name="text">
            <string>frequency domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_HY1_3">
        <property name="title">
         <string>Vibration #1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="1" colspan="2">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_VB1_ymin"/>
           </item>
           <item row="1" column="10">
            <widget class="QPushButton" name="pushButton_VB1_endPlot">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Plot:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="9">
            <widget class="QPushButton" name="pushButton_VB1_startPlot">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="11">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Record:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>tick</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_63">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_min</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_VB1_ymax"/>
           </item>
           <item row="1" column="6">
            <widget class="QLineEdit" name="lineEdit_VB1_tick"/>
           </item>
           <item row="1" column="13">
            <widget class="QPushButton" name="pushButton_VB1_endRec">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_64">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_max</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="pushButton_VB1_minmax">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="1" column="12">
            <widget class="QPushButton" name="pushButton_VB1_startRec">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="14">
            <widget class="QPushButton" name="pushButton_VB1_save">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="14">
            <widget class="QComboBox" name="comboBox_VB1"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_VB1_time" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget_VB1_freq" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_65">
           <property name="text">
            <string>time domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_66">
           <property name="text">
            <string>frequency domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_HY1_4">
        <property name="title">
         <string>Vibration #2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_17">
         <item row="0" column="1" colspan="2">
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox_HY1_8"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_67">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_min</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_HY1_ymin_8"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_68">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>y_max</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLineEdit" name="lineEdit_HY1_ymax_8"/>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="pushButton_HY1_minmax_8">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Plot:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="pushButton_HY1_startPlot_8">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QPushButton" name="pushButton_HY1_endPlot_8">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Record:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QPushButton" name="pushButton_HY1_startRec_8">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QPushButton" name="pushButton_HY1_endRec_8">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_HY1_time_8" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget_HY1_freq_8" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_69">
           <property name="text">
            <string>time domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_70">
           <property name="text">
            <string>frequency domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_ALL_endRec">
        <property name="text">
         <string>Stop Recording (for all sensors)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QPushButton" name="pushButton_checkSetUp">
        <property name="text">
         <string>CLICK ME if all sensors are ready!</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_ALL_startPlot">
        <property name="text">
         <string>Start Plotting (for all sensors)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_ALL_endPlot">
        <property name="text">
         <string>Stop Plotting (for all sensors)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_ALL_startRec">
        <property name="text">
         <string>Start Recording (for all sensors)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_ALL_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pipe Leak Detection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
